util/Migrator/DbMigrator.cs

```diff
@@ -23,10 +23,40 @@ public DbMigrator(string connectionString, ILogger<DbMigrator> logger)
         }.ConnectionString;
     }
 
-    public bool MigrateMsSqlDatabase(bool enableLogging = true,
+    public bool MigrateMsSqlDatabaseWithRetries(bool enableLogging = true,
         CancellationToken cancellationToken = default(CancellationToken))
     {
-        if (enableLogging && _logger != null)
+        var attempt = 1;
+
+        while (attempt < 10)
+        {
+            try
+            {
+                var success = MigrateDatabase(enableLogging, cancellationToken);
+                return success;
+            }
+            catch (SqlException e)
```

Comment:
ℹ️ My OCD says to call this variable `ex` ;)

Reply:
Fixed :)